test/images/pets/zookeeper-installer/install.sh

```diff
@@ -55,15 +55,15 @@ cp "${INSTALL_VOLUME}"/zookeeper/conf/zoo_sample.cfg "${INSTALL_VOLUME}"/zookeep
 
 # TODO: Should dynamic config be tied to the version?
 IFS="." read -ra RELEASE <<< "${VERSION}"
-if [ $(expr "${RELEASE[1]}") -gt 4 ]; then
+if [ "${RELEASE[1]}" -gt 4 ]; then
     echo zookeeper-"${VERSION}" supports dynamic reconfiguration, enabling it
-    echo "standaloneEnabled=false" >> "${INSTALL_VOLUME}"/zookeeper/conf/zoo.cfg
-    echo "dynamicConfigFile="${INSTALL_VOLUME}"/zookeeper/conf/zoo.cfg.dynamic" >> "${INSTALL_VOLUME}"/zookeeper/conf/zoo.cfg
+    echo "standaloneEnabled=false" >> "${INSTALL_VOLUME}/zookeeper/conf/zoo.cfg"
+    echo "dynamicConfigFile=${INSTALL_VOLUME}/zookeeper/conf/zoo.cfg.dynamic" >> "${INSTALL_VOLUME}/zookeeper/conf/zoo.cfg"
 fi
 
 # TODO: This is a hack, netcat is convenient to have in the zookeeper container
 # I want to avoid using a custom zookeeper image just for this. So copy it.
-NC=$(which nc)
+NC=$(command -v nc)
```

Comment:
For reference - [SC2230](https://github.com/koalaman/shellcheck/wiki/SC2230) - _which is non-standard. Use builtin 'command -v' instead_

Reply:
switched back to `which` per it being disabled in `hack/verify-shellcheck.sh`